activerecord/lib/active_record/model_schema.rb

```diff
@@ -431,6 +433,10 @@ def attribute_types # :nodoc:
         @attribute_types ||= Hash.new(Type.default_value)
       end
 
+      def after_load_schema(&block)
```

Comment:
MVP public interface, though it should really take and parse the same args as the usual callbacks so that it can call the args in the absence of a block